### PR TITLE
fix issue 19

### PR DESCRIPTION
**Pull request by filmakarov**

### CODE REVIEW

File: contracts/SmartSession.sol [hunk 1]
```diff
@@ -378,13 +378,13 @@ contract SmartSession is ISmartSession, SmartSessionBase, SmartSessionERC7739 {
...
         override
-        returns (bool valid)
+        returns (bool)
     {
         bytes32 contentHash = string(contents).hashERC7739Content();
         PermissionId permissionId = PermissionId.wrap(bytes32(signature[0:32]));
         signature = signature[32:];
         if (!$enabledERC7739Content[permissionId].contains(msg.sender, contentHash)) return false;
-        valid = $erc1271Policies.checkERC1271({
+        bool valid = $erc1271Policies.checkERC1271({
```
Comment (zeroknots) [inline contracts/SmartSession.sol:387]: isn't this worse on gas?

Comment (0xbok) [reply contracts/SmartSession.sol:426]: according to tests, it is increasing 4 units of gas

Comment (filmakarov) [reply contracts/SmartSession.sol:426]: thank you. I'll proceed with merging since it is more consistent

Comment (filmakarov) [reply contracts/SmartSession.sol:426]: thank you. I'll proceed with merging since it is more consistent

**Pull request merged**